.woodpecker.yml

```diff
@@ -1,18 +1,20 @@
 # TODO: The when: platform conditionals aren't working currently
 # See https://github.com/woodpecker-ci/woodpecker/issues/1677
 
+variables:
```

Comment:
https://woodpecker-ci.org/docs/next/usage/advanced-yaml-syntax

Reply:
Thanks! This is def helpful.